fix.js

```diff
@@ -145,6 +145,7 @@ jQuery.getScript(FixScript.dataset.layersUrl).done(function(){
 			addButton("Google Road Map", "googleroadmap");
 			addButton("Google Hybrid", "googlehybrid");
 			addButton("Google Terrain", "googleterrain");
+			addButton("Geoportail", "geoportail");
```

Comment:
This shouldn't be necessary, it's already done by the `AdditionalMapLayers.forEach(l => addButton(l.name, l.type))` line.